36/js/templateData.js

```diff
@@ -19,6 +19,6 @@ limitations under the License.
 var context={
     "version": "36",
     "dotVersion": "3.6",
-    "fullDotVersion": "3.6.1",
+    "fullDotVersion": "3.6.2-SNAPSHOT",
```

Comment:
shouldn't be snapshot, I should check out 3.6.1 and build from there I think

Reply:
Yes we want this to stay `3.6.1`